assets/src/domain/eventEditor/services/context/EntityListContext/DatetimesListContext.tsx

```diff
@@ -21,13 +22,29 @@ export const DatetimesListProvider: React.FC = ({ children }) => {
 	// memoize filter state
 	const filterState = useMemo(() => filters, [filtersStr]);
 
-	const filteredEntities = useFilteredEntities(domain, datesList, datetimes, filterState);
+	const { setSortBy, sortingEnabled } = filterState;
+
+	let filteredEntities = useFilteredEntities(domain, datesList, datetimes, filterState);
+
+	if (filterState.sortingEnabled) {
+		filteredEntities = notTrashed(filteredEntities);
+	}
```

Comment:
shouldn't this be:

```suggestion
	if (filterState.sortingEnabled) {
		filteredEntities = notTrashed(datetimes);
	}
```

because we want to use ALL datetimes when sorting and not the already filtered list?

Reply:
But we do need sorting by `order` 😄